src/index.html

```diff
@@ -1,13 +1,68 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="device-width initial-scale=1">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="page__content">
+    <main>
+      <section class="card" data-qa="card">
+        <img
+            src="https://github.com/NadieinOleh/layout_product-cards/blob/master/src/images/imac.jpeg?raw=true"
+            alt="imac"
+            class="card__imac-logo"
+        >
+
+        <div class="card__product">
+          <!-- <div class="imac"></div> -->
+          <div class="description">
```

Comment:
according to checklist you have to create BEM block for `card`. I would suggest you some changes for renaming here but check other places and fix them
![image](https://user-images.githubusercontent.com/22824431/221571007-f951a263-fe4a-4edc-8755-c5557dda6e8b.png)
```suggestion
          <div class="card__description">
```